src/serialport_win.cpp

```diff
@@ -77,7 +77,14 @@ void EIO_Open(uv_work_t* req) {
   }
 
   DCB dcb = { 0 };
+  SecureZeroMemory(&dcb, sizeof(DCB));
```

Comment:
I think the {0} should be OK, or we could use memset but the MSDN example used it so went with SecureZeroMemory